package.json

```diff
@@ -63,7 +63,7 @@
     "@elastic/react-search-ui-views": "1.4.1",
     "@elastic/search-ui": "1.4.1",
     "@jahia/i18next": "^1.0.0-beta.20180628145923",
-    "@jahia/search-ui-jahia-connector": "^1.0.0",
+    "@jahia/search-ui-jahia-connector": "../search-ui-jahia-connector",
```

Comment:
We need to put todo somewhere about this one so that we know why it doesn't build ;)

Reply:
I think this one is outdated, I changed to reference the one in npm registry.